micrometer-core/src/test/java/io/micrometer/core/aop/TimedAspectTest.java

```diff
@@ -518,7 +562,8 @@ void meterTagOnSuperClass() {
 
         enum AnnotatedTestClass {
 
-            CLASS_WITHOUT_INTERFACE(MeterTagClass.class), CLASS_WITH_INTERFACE(MeterTagClassChild.class);
+            // CLASS_WITHOUT_INTERFACE(MeterTagClass.class),
```

Comment:
This being commented out will result in tests for it not running. Actually it looks like all the tests under `MeterTagsTests` weren't being run because it wasn't annotated with `@Nested`.

Reply:
i've reenabled the enum value, must be an accidental commenting out((
All tests are executed as far as i can see

Reply:
> All tests are executed as far as i can see

It looks like it may just be an issue with running tests from my IDE (IntelliJ). I've confirmed when running the Gradle build, all the tests are being executed.